broker-cli/cmd/brokers.go

```diff
@@ -79,11 +84,27 @@ var (
 		Run: func(cmd *cobra.Command, args []string) {
 			flags.CheckFlags(&brokersFlags.project, &brokersFlags.broker)
 
-			http := httpAdapterFromFlag()
-			params := &adapter.DeleteBrokerParams{
-				BrokerURL: flags.ConstructBrokerURL(brokersFlags.host, brokersFlags.project, brokersFlags.broker),
+			client := httpAdapterFromFlag()
+			brokerURL := flags.ConstructBrokerURL(brokersFlags.host, brokersFlags.project, brokersFlags.broker)
+
+			if brokersFlags.cleanup == true {
+				err := cleanupBroker(client, brokerURL)
+				if err != nil {
+					if strings.Contains(err.Error(), userCancelledBrokerCleanup) {
```

Comment:
move this logic (throwing out interrupt from user, printing leftover instances) into cleanupBroker?

Reply:
can't be done, because I don't want "successfully cleaned up/ deleted broker" to be printed, instead I just want to return.

Reply:
okay. can you just move the printing of leftover instances?

Reply:
done.